plugins/identity-shiro/src/main/java/org/opensearch/identity/shiro/realm/OpenSearchRealm.java

```diff
@@ -25,8 +25,6 @@
 
 /**
  * Internal Realm is a custom realm using the internal OpenSearch IdP
- *
- * @opensearch.experimental
```

Comment:
What are the ramifications of removing this flag?

Reply:
Its not necessary in the plugins.

See https://github.com/opensearch-project/OpenSearch/pull/14630#discussion_r1722171057